web/templates/settings/integration_events.hbs

```diff
@@ -0,0 +1,13 @@
+{{#each events }}
+    <div class="integration-event-box">
```

Comment:
I would rename this class to "integration-event-wrapper".

Reply:
done